manifests/host.pp

```diff
@@ -1,17 +1,17 @@
 # == Define: dhcp::host
 #
 define dhcp::host (
-  Stdlib::IP::Address $ip,
+  Optional[Stdlib::Compat::Ip_address] $ip = undef,
```

Comment:
Looks like this is undoing some of the tidying from https://github.com/voxpupuli/puppet-dhcp/pull/217
Could you change this to `Optional[Stdlib::IP::Address] $ip = undef`?

Reply:
I will update this and merge it afterwards